### PR TITLE
TW-400 Webhooks doc refresh

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/pages/docs/integrations/webhooks.md [hunk 1]
```diff
@@ -19,66 +17,57 @@ contextual_links:
     url:  "https://www.postman.com/postman/workspace/4be86d9c-6576-4369-b74f-43991df7a4bd"
 ---
 
-Postman provides a custom webhook integration which enables you to automate workflows between your favorite apps and services to get notifications, synchronize files, collect data, and more. It offers many services with predefined flows available for easy implementation.
+Postman's custom webhook integration enables you to automate workflows between your favorite apps and services to get notifications, synchronize files, collect data, and more. It offers services with predefined flows available for easy implementation.
 
-You can configure a custom webhook with Postman to send events such as monitor results, team and collection-specific activity feeds, and to back up your Postman Collections.
+You can configure a custom webhook with Postman to send events such as monitor results or team and collection-specific activity feeds, or to back up your Postman Collections.
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/webhooks.md:22]: Should `Collections` be capitalized here? 🤔

File: src/pages/docs/integrations/webhooks.md [hunk 2]
```diff
@@ -19,66 +17,57 @@ contextual_links:
...
-* [Configuring custom webhook URL](#configuring-custom-webhook-url)
-* [Back up your Postman Collections](#back-up-your-postman-collections)
-    * [Backup Collections](#backup-collections)
-* [Send collection activity feed to custom webhooks](#send-collection-activity-feed-to-custom-webhooks)
-* [Send Monitor run results to custom webhooks](#send-monitor-run-results-to-custom-webhooks)
-    * [Monitor Run Results](#monitor-run-results)
+* [Enabling static IP support](#enabling-static-ip-support)
+* [Configuring custom webhooks](#configuring-custom-webhooks)
+* [Send updates for a Postman Collection to a custom webhook](#send-updates-for-a-postman-collection-to-a-custom-webhook)
+* [Send monitor results to a custom webhook](#send-monitor-results-to-a-custom-webhooks)
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/webhooks.md:29]:
```suggestion
* [Send monitor results to a custom webhook](#send-monitor-results-to-a-custom-webhook)
```

File: src/pages/docs/integrations/webhooks.md [hunk 3]
```diff
@@ -19,66 +17,57 @@ contextual_links:
...
 
-[![custom_webhook](https://assets.postman.com/postman-docs/custom-webhooks.jpg)](https://assets.postman.com/postman-docs/custom-webhooks.jpg)
+   [![custom_webhook](https://assets.postman.com/postman-docs/custom-webhooks.jpg)](https://assets.postman.com/postman-docs/custom-webhooks.jpg)
 
-Each integration's page explains how to use the integration and what it can do. If available, you can view previously configured integrations for the selected integration.
+   The integration's page has choices for each type of custom webhook. If available, you can view previously configured integrations for the selected integration.
 
-![Workspace Integrations](https://assets.postman.com/postman-docs/webhooks-teammates.jpg)
-2. Select __Add Integration__ to configure your integration. Enter the required information for account and access authorization. Select the workspace you need to add the integration to and proceed with the integration setup.
+   ![Webhooks integration choices](https://assets.postman.com/postman-docs/webhook-integrations.jpg)
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/webhooks.md:50]:
```suggestion
   ![Webhooks integration choices](https://assets.postman.com/postman-docs/webhook-integrations.jpg)
   
```

Comment (HeatherStenson) [reply src/pages/docs/integrations/webhooks.md:50]: Spacing around the screenshot

File: src/pages/docs/integrations/webhooks.md [hunk 4]
```diff
@@ -19,66 +17,57 @@ contextual_links:
...
-[![webhooks collections1](https://assets.postman.com/postman-docs/add-integration-setup.jpg)](https://assets.postman.com/postman-docs/add-integration-setup.jpg)
+### Send updates for a Postman Collection to a custom webhook
 
-Once the integration has been created, you can view the integration you just created in addition to the other active integrations created by your team:
+To send updates for a Postman Collection to a custom webhook:
 
-[![configured integrations](https://assets.postman.com/postman-docs/view-webhooks-all.jpg)](https://assets.postman.com/postman-docs/view-webhooks-all.jpg)
+1. In the **Webhooks Integrations** page, next to **Backup a collection**, select **Add Integration**.
+1. In the **Add integration** page:
+   * Enter any name for the nickname.
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/webhooks.md:63]:
```suggestion
   * Enter a **Nickname** for the integration.
```

File: src/pages/docs/integrations/webhooks.md [hunk 5]
```diff
@@ -19,66 +17,57 @@ contextual_links:
...
+   * Select the collection you wish to back up.
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/webhooks.md:64]:
```suggestion
   * Under **Choose Collection**, select the collection you wish to send updates for.
```

File: src/pages/docs/integrations/webhooks.md [hunk 6]
```diff
@@ -96,33 +85,25 @@ The following is a schema for Backup Collections:
...
 
 Postman Monitors allows you to run your collections on a schedule without any manual intervention. With the custom webhooks, you can use those results by connecting to other available services.
 
 To send monitor run results to custom webhooks:
 
-1. Select **Add Integration**.
-2. In the **Monitor Run Results** page, select the monitor you want to send to custom webhooks.
+1. In the **Webhooks Integrations** page, next to **Post monitoring results**, select **Add Integration**.
+2. In the **Add integration** page:
+   * Enter any name for the nickname.
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/webhooks.md:96]:
```suggestion
   * Enter a **Nickname** for the integration.
```

File: src/pages/docs/integrations/webhooks.md [hunk 7]
```diff
@@ -201,19 +182,19 @@ The following is a schema for monitor run results:
...
-The activity feed is where you can track changes made to your collections and within your team. Integrating with webhooks gives you the freedom to connect with many services.
+The activity feed is where you can track changes made to your collections and within your team.
 
 To send a team activity feed to a custom webhook:
 
-1. Select **Add Integration**.
-2. In the **Team Activity Feed** page, enter the webhook URL to send team updates to this specific URL.
+1. In the **Webhooks Integrations** page, next to **Post team activity**, select **Add Integration**.
+2. In the **Add integration** page:
+   * Enter any name for the nickname.
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/webhooks.md:191]:
```suggestion
   * Enter a **Nickname** for the integration.
```

**Review by HeatherStenson — APPROVED** [review]
Looks good! A few suggestions, no blockers. 🍊

File: src/pages/docs/integrations/webhooks.md (hunk 1 above)
Comment (jkonrath-postman) [reply src/pages/docs/integrations/webhooks.md:22]: It's a branded term, but we need to talk about how much we should be doing this.

File: src/pages/docs/integrations/webhooks.md (hunk 3 above)
Comment (jkonrath-postman) [reply src/pages/docs/integrations/webhooks.md:50]: Oops, that suggestion broke lint :(

File: src/pages/docs/integrations/webhooks.md [hunk 8]
```diff
@@ -19,66 +17,58 @@ contextual_links:
...
-* [Configuring custom webhook URL](#configuring-custom-webhook-url)
-* [Back up your Postman Collections](#back-up-your-postman-collections)
-    * [Backup Collections](#backup-collections)
-* [Send collection activity feed to custom webhooks](#send-collection-activity-feed-to-custom-webhooks)
-* [Send Monitor run results to custom webhooks](#send-monitor-run-results-to-custom-webhooks)
-    * [Monitor Run Results](#monitor-run-results)
+* [Enabling static IP support](#enabling-static-ip-support)
+* [Configuring custom webhooks](#configuring-custom-webhooks)
+* [Send updates for a Postman Collection to a custom webhook](#send-updates-for-a-postman-collection-to-a-custom-webhook)
+* [Send monitor results to a custom webhook](#send-monitor-results-to-a-custom-webhook)
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/webhooks.md:29]:
```suggestion
* [Send monitor results to custom webhooks](#send-monitor-results-to-custom-webhooks)
```

File: src/pages/docs/integrations/webhooks.md [hunk 9]
```diff
@@ -96,33 +86,25 @@ The following is a schema for Backup Collections:
...
-To send collection activity feed to custom webhooks:
-
-1. Select **Add Integration**.
-2. In the **Collection Activity Feed** page, enter the webhook URL to send team updates to this specific URL.
-3. Select **Add Integration**.
-
-## Send Monitor run results to custom webhooks
+## Send monitor results to custom webhooks
 
 Postman Monitors allows you to run your collections on a schedule without any manual intervention. With the custom webhooks, you can use those results by connecting to other available services.
```
Comment (mark-anthony-postman) [inline src/pages/docs/integrations/webhooks.md:91]:
```suggestion
Postman Monitors enable you to run your collections on a schedule without any manual intervention. With the custom webhooks, you can use those results by connecting to other available services.
```

**Review by mark-anthony-postman — APPROVED** [review]
LGTM

File: src/pages/docs/integrations/webhooks.md (hunk 3 above)
Comment (HeatherStenson) [reply src/pages/docs/integrations/webhooks.md:50]: 🤯

**Pull request merged**